apps/api/package.json

```diff
@@ -99,6 +99,7 @@
     "passport-headerapikey": "^1.2.2",
     "passport-jwt": "^4.0.1",
     "passport-oauth2": "^1.8.0",
+    "prettier": "~3.3.3",
```

Comment:
prettier is used to format the code when switching the editor from blocks to html